### DISCUSSION TITLE
Flexipatch compiled successfully but doesn't show up in Ly

### DISCUSSION THREAD

**Discussion by abneb** [bakkeby/dwm-flexipatch#320] [Q&A]
A copy of my xinit
![image](https://user-images.githubusercontent.com/69637662/210630187-53dd4a91-402c-4a29-a011-7293c8982e48.png)

 And in config.mk I didn't change anything

**Comment by bakkeby** [comment]
This should be due to #317.

**Comment by monosans** [comment]
Are you sure you installed it? Maybe you compiled it, but did not move it to the folder with binary files. To install run:

```bash
sudo make install
```

**Reply by abneb** [reply]
Yep man, I've installed
And double checked it with which dwm

**Reply by monosans** [reply]
> Yep man, I've installed And double checked it with which dwm

Then try this:

```bash
startx $(which dwm)
```

**Comment by abneb** [comment]
That initialize dwm but only when I execute this command. When I logout it keeps missing in Ly.

**Reply by monosans** [reply]
> That initialize dwm but only when I execute this command. When I logout it keeps missing in Ly.

Try to apply the patch from [here](https://github.com/bakkeby/dwm-flexipatch/issues/317#issuecomment-1366599819) and reinstall dwm (make sure to `make uninstall` the old one first).

**Comment by bakkeby** [comment]
If you have a desktop entry like `/usr/share/xsessions/dwm.desktop` then I'd assume that `ly` will be able to find that.

**Comment by abneb** [comment]
I'm making a terrible mistake cuz I was doing this in my production machine. But I have the Xsessions entry ok. It still doesn't work. I'll clone my setup in  a VM to try out with safety.

**Comment by bakkeby** [comment]
@carlosonrails can you tell us what is the output of the following command on your machine then?

```sh
ls -l /usr/share/xsessions/dwm.desktop
```

**Reply by abneb** [reply]
yep. Right now I remember that I had configure it timeshift so a restore to my previous build of Dwm (without flexipatch) and keep my Flexipatch saved for tomorrow. 

![image](https://user-images.githubusercontent.com/69637662/211005786-b5553c7e-1674-4207-8455-91bb4d9d09d2.png)

**Reply by bakkeby** [reply]
That's not the command I asked you to run though.

**Reply by abneb** [reply]
Sorry. Here it is

![image](https://user-images.githubusercontent.com/69637662/211170679-a05738be-059c-4bee-8b12-368cbaf88fe4.png)

**Reply by bakkeby** [reply]
I don't see any good reason why `ly` would not read that file. As far as I can tell the file location, it's content, user ownership and permissions all look fine.

I did notice that it reads from only one directory (/usr/share/xsessions/ by default), so if the .desktop file is placed in /usr/local/share/xsessions/ then `ly` won't find it. I doubt that you've changed any configuration in this respect though.

**Reply by ghost** [reply]
I tried using sddm (because I'm familiar with it) and dwm-flexipatch, it works perfectly so I think this is a `ly` issue.

**Reply by abneb** [reply]
Ly was the issue. I was looking if that happens with Chadmw and have the same error. Like @speediegq I change to sddm and everything works fine. 
I think the could be closed. 
Thanks U all
